appinfo/info.xml

```diff
@@ -11,7 +11,7 @@
 - **💾 Open format:** Files are saved as [Markdown](https://en.wikipedia.org/wiki/Markdown), so you can edit them from any other text app too.
 - **✊ Strong foundation:** We use [🐈 tiptap](https://tiptap.scrumpy.io) which is based on [🦉 ProseMirror](https://prosemirror.net) – huge thanks to them!
 	]]></description>
-	<version>4.0.0</version>
+	<version>4.1.0</version>
```

Comment:
We only bumped for main do far, but since it's an internal version anyways fine with me

Reply:
Mh, not sure I understand. You mean to bump it to `5.0.0` instead?